stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2856,9 +2919,8 @@ internal struct _Native${Self}Storage<${TypeParameters}> {
     }
   }
 
-  /// The number of _logical elements_ the ${a_self} can store. Not to be
-  /// confused with `capacity`, the total number of raw `UInt8` bytes stored
-  /// by the `ManagedBuffer`.
+  // Forwarding the individual fields of the buffer in various forms
```

Comment:
Consider using a swift-divider for these section-division comments.  Look for `//===---` in the source for examples